
Dataset loaded, beginning training
Traceback (most recent call last):
  File "neurals/scripts/train.py", line 136, in <module>
    main()
  File "neurals/scripts/train.py", line 48, in main
    for i, data in enumerate(train_loader):
  File "/home/sirius/anaconda3/envs/nimble_dev/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 652, in __next__
    data = self._next_data()
  File "/home/sirius/anaconda3/envs/nimble_dev/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 692, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/sirius/anaconda3/envs/nimble_dev/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/sirius/anaconda3/envs/nimble_dev/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/sirius/anaconda3/envs/nimble_dev/lib/python3.8/site-packages/torch/utils/data/dataset.py", line 290, in __getitem__
    return self.dataset[self.indices[idx]]
  File "/home/sirius/sirui/contact_planning_dexterous_hand/neurals/dataset.py", line 164, in __getitem__
    ans["label"] = self.labels[idx]
